src/npeg/parsepatt.nim

```diff
@@ -188,19 +188,21 @@ proc parseGrammar*(ns: NimNode, dot: Dot=nil, dumpRailroad = true): Grammar =
 
     if n.kind == nnkInfix and n[0].eqIdent("<-"):
 
-      if n[1].kind in { nnkIdent, nnkDotExpr}:
-        let name = n[1].repr
+      case n[1].kind
+      of nnkIdent, nnkDotExpr, nnkPrefix:
+        let name = if n[1].kind == nnkPrefix: expectIdent n[1][0], ">"; n[1][1].repr
```

Comment:
`expectIdent` is defined as:

`proc expectIdent*(n: NimNode, name: string) {.compileTime, since: (1,1).} =`

and it doesn't exist in Nim 1.0.x, which means that `npeg` is now broken on Nim 1.0.x :/

@Clyybber @zevv